jenkins/scripts/slurm_run.sh

```
@@ -87,7 +78,24 @@ export LD_LIBRARY_PATH=$containerLDLibPath
 echo "Library Path:"
 echo "$LD_LIBRARY_PATH"
 env | sort
-fullCmd="${testCmdLines[*]}"
 echo "Running: $testCase"
-echo "Full Command: $fullCmd"
-eval $fullCmd
+echo "Full Command: $pytestCommand"
+eval $pytestCommand
+
+if [ "$perfMode" = "true" ]; then
```

Shellcheck (0.10.0)

[warning] perfMode is referenced but not assigned.

(SC2154)